pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -226,8 +226,10 @@ void processArguments() throws Exception {
         protected String inputs;
         @Parameter(names = "--topicsPattern", description = "TopicsPattern to consume from list of topics under a namespace that match the pattern. [--input] and [--topicsPattern] are mutually exclusive. Add SerDe class name for a pattern in --customSerdeInputs (supported for java fun only)")
         protected String topicsPattern;
-        @Parameter(names = "--output", description = "The function's output topic")
+        @Parameter(names = "--output", description = "The function's output topic (use skipOutput flag to skip output topic)")
         protected String output;
+        @Parameter(names = "--skipOutput", description = "Skip publishing function output to output topic")
```

Comment:
can you rename it to `--skip-output`? we are attempting to standardize the naming convention in the command line - "lower cases with '-' as separators".

Reply:
yes.. function cli was not consistent with other pulsar-cli. will rename it.